website/scripts/build-static-content.js

```diff
@@ -625,7 +625,7 @@ module.exports = {
           ).replace(/[^a-z0-9\-]/ig,'');
 
           // Determine the rootRelativeUrlPath for this open position, this will be used as the page's URL and to check if a markdown page already exists with this page's URL
-          let rootRelativeUrlPath = '/handbook/company/'+encodeURIComponent(_.kebabCase(openPosition.jobTitle));
+          let rootRelativeUrlPath = '/handbook/company/open-positions/'+encodeURIComponent(_.kebabCase(openPosition.jobTitle));
```

Comment:
What about redirects?

Reply:
Up to Sam and Eric whether to yolo

Reply:
@mikermcneil 👍 I added redirects for those positions